resources/home/dnanexus/generate_gcnv_bed.py

```
@@ -15,103 +24,172 @@ def parse_args():
+    try:
         file_df = pd.read_csv(
-            file, sep='\t', comment="@", header=0,
-            names=['chr', 'start', 'end', sample_name]
+            copy_ratio_file,
+            sep="\t",
+            comment="@",
+            header=0,
+            names=["chr", "start", "end", sample_name],
         )
+    except pd.errors.EmptyDataError:
+        raise ValueError(f"No data found in file: {copy_ratio_file}")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)